pkg/init/cmd/service/prepare.go

```diff
@@ -122,7 +118,27 @@ func parseMountOptions(options []string) (int, string) {
 	return flag, strings.Join(data, ",")
 }
 
-// prepareFilesystem sets up the mounts, before the container is created
+// newCgroup creates a cgroup (ie directory) under all directories in /sys/fs/cgroup
+// we could use github.com/containerd/cgroups but it has a lot of deps and this is just a sugary mkdir
+func newCgroup(cgroup string) error {
+	dirs, err := ioutil.ReadDir("/sys/fs/cgroup")
```

Comment:
The docs weirdly don't say, but I suppose this omits `.` and `..`?

Reply:
yeah it does... not sure how you get them if you need them, use `x/sys/unix` I guess...